app/service/cdo.js

```diff
@@ -300,38 +300,15 @@ class CdoService {
       indexNumber,
       cdoTaskList.dog.name,
       payload.microchipNumber,
-      payload.dogNotFitForMicrochip, // unfit - boolean (from payload?)
-      microchipDate, // microchip_date (from payload?) - as string object not date
-      payload.neuteringConfirmation, // neutering date (from payload?) - as string object not date (or empty string),
-      payload.dogNotNeutered, // under16 (from payload)
-      user.username // user.username
+      payload.dogNotFitForMicrochip,
+      microchipDate,
+      payload.neuteringConfirmation,
```

Comment:
Are you supplying the date here as either formatted as say DD/MM/YYYY or as a blank string '' if not applicable? The email template expects that content